src/features/quick-search/QuickSearchMessage.js

```diff
@@ -1,30 +1,24 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+// @flow
+import * as React from 'react';
 import classNames from 'classnames';
 
 import './QuickSearchMessage.scss';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class QuickSearchMessage extends Component {
-    static propTypes = {
-        children: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
-        isShown: PropTypes.bool.isRequired,
-    };
+type Props = {
+    /** The message to string or elements to show */
+    children: string | React.Node,
+    /** Boolean to indicate if the message should be shown */
+    isShown: boolean,
+};
 
-    static defaultProps = {
-        isShown: false,
-    };
+const QuickSearchMessage = ({ children, isShown = false }: Props) => {
```

Comment:
Use QuickSearchMessage.defaultProps = {} at the bottom instead in inlining it

Reply:
done